<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CClient</class>
 <widget class="QMainWindow" name="CClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CClient</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>642</width>
      <height>722</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="btnConnectServer">
         <property name="text">
          <string>连接服务器</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="filePostion"/>
           </item>
           <item>
            <widget class="QPushButton" name="btnSelectFile">
             <property name="text">
              <string>选择文件</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnTransfer">
             <property name="text">
              <string>上传文件</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QProgressBar" name="barProgress">
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnCancel">
             <property name="text">
              <string>取消上传</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="labelImgDisplay">
       <property name="minimumSize">
        <size>
         <width>640</width>
         <height>640</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
